src/main/java/example/CourierClientApp.java

```diff
@@ -110,6 +110,22 @@ public static void main(String[] args) throws IOException {
         SendEnhancedResponseBody sendEnhancedResponseBodyMeta = new SendService().sendEnhancedMessage(sendEnhancedRequestBodyMeta);
         System.out.println(sendEnhancedResponseBodyMeta);
 
+        // Enhanced Send - timeout
+        Gson gsonTimeout = new Gson();
+        SendEnhancedRequestBody sendEnhancedRequestBodyTimeout = new SendEnhancedRequestBody();
+        SendRequestMessage sendRequestMessageTimeout = new SendRequestMessage();
+        HashMap<String, Object> toMapTimeout = gsonTimeout.fromJson("{'email':'tejas@courier.com'}", HashMap.class);
+
+        HashMap<String, Object> timeout = gsonTimeout.fromJson("{'timeout':{'message':360000000}}", HashMap.class);
```

Comment:
@lednax can we double check this example works? I am assuming this would do timeout.timeout.message = 360000000

Reply:
Good catch! (My dev env for Java is so broken 💀 )